### ISSUE TITLE
 Focus is lost when the quantity is increased / decreased by entering number into the quantity stepper

### ISSUE THREAD

**Issue by Larisa-Staroverova** [SAP/spartacus#11556]

**Comment by Uli-Tiger** [comment]
adding focus directive to item-counter does not work.
adding  focus directive to the the input field inside the item-counter does work.
asked in slack channel for advice: https://sap-cx.slack.com/archives/CMBTFUYVA/p1616154609018900

**Comment by Uli-Tiger** [comment]
we should revisit this after https://github.com/SAP/spartacus/issues/9531 is solved and https://github.com/SAP/spartacus/pull/10507 is merged

**Comment by Uli-Tiger** [comment]
checked the referenced ticket. I do not see how the refresh focus feature helps us.

**Comment by Uli-Tiger** [comment]
asked via slack for advice https://sap-cx.slack.com/archives/CMBTFUYVA/p1619186176175800?thread_ts=1616154609.018900&cid=CMBTFUYVA

**Comment by Uli-Tiger** [comment]
maybe we should just make the necessary changes ourself and create a PR for it
also consider DDLB.

**Comment by giancorderoortiz** [comment]
This is a core Spartacus issue. 
Please check with @developpeurweb and/or @hackergil. 
Same situation happens in cart page electronics or b2b.
